content/OTP/Guides/OTP_Walk-through.adoc

```diff
@@ -222,7 +226,7 @@ During authentication:
 
 For two-factor authentication, add a field or other means to enter credentials. Indicate that the YubiKey must also be inserted and tab-touched.
 
-*Step 2* Pass the OTP to your  client.
+*Step 2* Pass the OTP to your YubiCloud client.
 
 Your YubiCloud client validates the OTP in the YubiCloud. The YubiCloud validation server returns a response.
```

Comment:
Chris Halos: Your YubiCloud client validates the OTP in the YubiCloud = against/with the YubiCloud?